demo/deploy-bookstore-with-same-sa.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
I see potential to consolidate the bookstore spec files and just use a variable to specify the service account.

Reply:
good suggestion, let me address it in a future PR